integration-tests/utils/integration-helpers.js

```diff
@@ -201,7 +264,14 @@ const enableDataCatalog = async (page) => {
 
 const clickNavChildAndLoad = async (page, tab) => {
   // click triggers a page navigation event
-  await Promise.all([page.waitForNavigation(navOptionNetworkIdle()), noSpinnersAfter(page, { action: () => click(page, navChild(tab)) })]);
+  await Promise.all([
+    page.waitForNavigation(navOptionNetworkIdle(Millis.ofMinute)),
```

Comment:
A minute wait here seems very long. If it is necessary, can you add a comment about why?

Reply:
This is the default value; I meant to remove the explicit parameter here but it was set to a minute before I touched it